internal/cmd/local/command_kafka.go

```diff
@@ -0,0 +1,30 @@
+package local
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+)
+
+const imageName = "523370736235.dkr.ecr.us-west-2.amazonaws.com/confluentinc/kafka-local:latest"
+const urlPrefix = "http://0.0.0.0:%s"
```

Comment:
What is this the URL for? Make this variable name more descriptive if it's global.

Reply:
it's for local communication of krest, but might be used later by sr too. `localhostPrefix`?

Reply:
Ah, cool. Can you change it to "http://localhost:%d" maybe? (or better yet, use the `url` package!)